app/push_notifications/push_notifications.android.js

```diff
@@ -70,17 +70,17 @@ class PushNotification {
 
         if (options.popInitialNotification) {
             PendingNotifications.getInitialNotification().
```

Comment:
We should probably standardize this as
```
action().then(() => {
    ...
}).catch((err) => {
    ...
}
```
since I think that's how its done on the web app. That's a future thing though